CHANGELOG.md

```diff
@@ -4,6 +4,20 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog 1.0.0].
 
+## v28.2.0 (2024-12-17)
+
+### Feat
+
+- **FCL-532**: assign FCLIDs on document publication
+- **FCL-532**: add ability to retrieve identifiers by type
+- **FCL-499**: add new FCLID identifier class
+- **FCL-499**: add method to get next sequence number from MarkLogic
+
+### Fix
```

Comment:
Surprised we're explicitly tracking the dependencies in the changelog, but not a problem.